src/v2/components/Preferences.jsx

```diff
@@ -0,0 +1,176 @@
+// -----------------------------------------------------------------------------
+// Copyright 2022 Open Climate Tech Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+import React, {useEffect, useRef} from 'react'
+
+import Duration from '../modules/Duration.mjs'
+import query from '../modules/query.mjs'
+
+const Props = {
+  CENTER: [34.3987, -119.8199], // Santa Barabara, CA.
```

Comment:
Maybe add a comment that this is initial value to be ignored once API response arrives

Reply:
Done